### ISSUE TITLE
Zipper, Apex implementation for zip

### ISSUE THREAD

**Issue by thvd** [certinia/apex-zip#2]
Did you see this Apex implementation of Zip: https://github.com/pdalcol/Zipper? 

Maybe it's a good idea to mention it in the [README.markdown](https://github.com/financialforcedev/apex-zip/blob/master/README.markdown) So people know that there is a proper Apex implementation, without client side requirements.

**Comment by afawcett** [comment]
Yes i have just seen this library, most interesting. I have asked a question to the author about the Apex CPU consumption of it though.
